### PR TITLE
Added google map

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

**Comment by arash-fs** [comment]
Please check that the links should open in the same tab or on a new tab.

**Comment by sara-fs** [comment]
@arash-fs  In my opinion, It's better that we let the UI and QA to check these things, since we pass the card to UI review and QA and they give their opinion if it is important to them. Better not to waste time more than this on design.

File: src/pages/contact-us/_offices.js [hunk 1]
```diff
@@ -209,56 +252,77 @@ export const Offices = () => {
...
+                                    color="black-3"
+                                    size="var(--text-size-s)"
+                                    href="https://maps.google.com/?q=Edificio+Australia+Asunción&-25.282569,-57.563805&ll=-25.282569,-57.563805&z=17"
+                                >
                                     {localize(
                                         '886, Edificio Australia, Calle Herib, Campos Cervera',
                                     )}
-                                </StyledText>
-                                <StyledText textIndent="1.5em" marginBttom="0">
+                                    <br />
```
Comment (mustofa-binary) [inline src/pages/contact-us/_offices.js]: can use `Localize` with components instead

File: src/pages/contact-us/_offices.js [hunk 2]
```diff
@@ -209,56 +252,77 @@ export const Offices = () => {
...
                         <ImageWrapper>
-                            <Image
-                                img_name="map-paraguay.png"
-                                alt={localize('paraguay Office')}
-                                width="49rem"
-                                height="100%"
-                            />
+                            <MapLink
+                                target="_blank"
+                                href="https://maps.google.com/?q=Edificio+Australia+Asunción&-25.282569,-57.563805&ll=-25.282569,-57.563805&z=17"
```
Comment (mustofa-binary) [inline src/pages/contact-us/_offices.js]: google maps is totally foreign url, so can use rel='noopener noreferrer'

File: src/pages/contact-us/_offices.js [hunk 3]
```diff
@@ -115,27 +123,39 @@ export const Offices = () => {
...
+                                <MapIconWrapper>
                                     <MapIcon />
+                                </MapIconWrapper>
+                                <LinkText
+                                    target="_blank"
+                                    color="black-3"
+                                    size="var(--text-size-s)"
+                                    href="https://maps.google.com/?q=Binary+(Europe)+Ltd&35.903605,14.460732&ll=35.903605,14.460732&z=17"
+                                >
                                     {localize('Level 3, W Business Centre,')}
```
Comment (mustofa-binary) [inline src/pages/contact-us/_offices.js]: can use `Localize` with components

**Pull request merged**